tester.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+version="V0.0.3"
+usage="$(basename "$0") [-help] [-v] -- Simple test suite for dfshack
+
+where:
+    -h,   show this help text
+    -v,   Shows version information"
+
+###############################################################################
+#Function: auto_testing                                                       #
+#Prupose: Runs auto test case for dfshack by creating severle files changing  #
+#         their permissions and moving them around                            #
+#arguments: none                                                              #
+#return: none                                                                 #
+###############################################################################
+function auto_testing {
+  echo -e "\033[1;33mAutomated dfshack test\033[0m"
+ 
+  echo -e "\033[1;35mCreating folder outer\033[0m"
+  file="/nethome/users/$USER/outer"
+  mkdir "$file"
+  ensure $file
```

Comment:
It would be good to ensure the file doesn't exist before this test runs or to delete it before running the test. That way you don't get false positives!